src/main/python/net/i2cat/cnsmo/manager/cnsmo.py

```diff
@@ -95,8 +96,8 @@ def register_service(self, service):
         :return:
         """
         self.__system_state_manager.save(service)
-        print "saved"
-        print service.get_service_id()
+        print("saved")
```

Comment:
Haha, no. The project uses Python 2.7.6.
However, in this version `print` is also a function. I just changed it to increase compatibility, just in case we move to Python 3 some day  :P